shared/teams/team/members/member-row/index.js

```diff
@@ -77,56 +79,71 @@ export const TeamMemberRow = (props: Props) => {
       ? 'Has reset their account'
       : 'Has reset their account; admins can re-invite'
   }
+
   return (
-    <Box
-      style={{
-        ...globalStyles.flexBoxRow,
-        alignItems: 'center',
-        flexShrink: 0,
-        height: isMobile ? 56 : 48,
-        padding: globalMargins.tiny,
-        width: '100%',
-      }}
-    >
-      <ClickableBox
-        style={{...globalStyles.flexBoxRow, flexGrow: 1, alignItems: 'center'}}
-        onClick={active ? onClick : undefined}
+    <Box style={active ? stylesContainer : stylesContainerReset}>
+      <Box
+        style={{
+          ...globalStyles.flexBoxRow,
+          alignItems: 'center',
+          flexShrink: 0,
+          height: isMobile ? 56 : 48,
+          padding: globalMargins.tiny,
+          width: '100%',
+        }}
       >
-        <Avatar username={username} size={isMobile ? 48 : 32} />
-        <Box style={{...globalStyles.flexBoxColumn, marginLeft: globalMargins.small}}>
-          <Box style={globalStyles.flexBoxRow}>
-            <Usernames
-              type="BodySemibold"
-              colorFollowing={true}
-              users={[{username, following, you: you === username}]}
-            />
-            {!active && (
-              <Meta
```

Comment:
Also: android studio complained about this, so I changed it from Meta to Text

Reply:
Meta isn't text